java/client/src/org/openqa/selenium/support/ui/WebDriverWait.java

```diff
@@ -70,8 +69,8 @@ public WebDriverWait(WebDriver driver, long timeOutInSeconds, long sleepInMillis
   public WebDriverWait(WebDriver driver, Clock clock, Sleeper sleeper, long timeOutInSeconds,
       long sleepTimeOut) {
     super(driver, clock, sleeper);
-    withTimeout(timeOutInSeconds, TimeUnit.SECONDS);
-    pollingEvery(sleepTimeOut, TimeUnit.MILLISECONDS);
+    withTimeout(timeOutInSeconds, ChronoUnit.SECONDS);
```

Comment:
Since we're keeping the public API stable, we don't need these changes.

Reply:
but API becomes deprecated, shouldn't we avoid using deprecated API?

Reply:
Only if we deprecate the methods.